### ISSUE TITLE
401 unauthorized when I start a room

### ISSUE THREAD

**Issue by bpetit** [bigbluebutton/bbb-install#178]
Hi,

Just made a fresh install from bbb-install with greenlight. Worked perfectly.

Then I rebooted the server and once back, when I try to start a room, I have this page, after a waiting page with 3 dots:

![screenshot_7](https://user-images.githubusercontent.com/906428/80285306-2f475d80-8724-11ea-8699-aecc01b6febf.png)

I'm admin on greenlight.

Any idea ? 

Thanks

**Comment by feschoppe** [comment]
I got the exact same issue after a reboot of my server. I made a fresh install last week and today I made a reboot and then I also got the screen 401 when starting a room.

**Comment by bpetit** [comment]
I was able to make greenlight working again by rerunning the install process with bbb-install. This feels like a dirty workaround and obviously is not the solution, but just to mention.

**Comment by jakiro2017** [comment]
> I was able to make greenlight working again by rerunning the install process with bbb-install. This feels like a dirty workaround and obviously is not the solution, but just to mention.

Did the reinstall upgrade bbb as well?

**Comment by bleaktradition** [comment]
I had the same issue but it disappeared after I disabled unattended upgrades on Ubuntu.
Someone mentioned it on a different page but I don't remember where.

**Comment by bpetit** [comment]
> > I was able to make greenlight working again by rerunning the install process with bbb-install. This feels like a dirty workaround and obviously is not the solution, but just to mention.
> 
> Did the reinstall upgrade bbb as well?

I didn't check the exact version the first time sorry.

**Comment by IncubusRK** [comment]
You can use workaround from issue #9466

**Comment by borhanreo** [comment]
I have the problem 
![image](https://user-images.githubusercontent.com/11036088/100700140-addc3b80-3394-11eb-88cb-27c3dd77acee.png)

**Comment by AliDjango** [comment]
same just installed BBB and this is what i get from fresh installation

**Comment by JosebaNr** [comment]
Same issue for me, a fresh install of bionic-230-dev on a fresh Ubuntu 18.04.5 VM with the BBB API demo yields this error when accessing any room. Reconnecting/re-entering the session doesn't have any effect

**Comment by borhanreo** [comment]
Need to add domain/subdomain name https://xyz.xyz.com

**Comment by JosebaNr** [comment]
> Need to add domain/subdomain name https://xyz.xyz.com

You mean that domain/subdomain properties are required to have a working environment? Doesn't it work by omitting that property and using local IP addresses? I want to setup a local development environment to develop html5 client changes and I don't really need nor want to register a FQDN to work locally.

**Comment by borhanreo** [comment]
> > Need to add domain/subdomain name https://xyz.xyz.com
> 
> You mean that domain/subdomain properties are required to have a working environment? Doesn't it work by omitting that property and using local IP addresses? I want to setup a local development environment to develop html5 client changes and I don't really need nor want to register a FQDN to work locally.

You can. But i was faced the same issue in local. when i use aws ec2 machine and configure domain with ssl certificate then bbb work properly my end

**Comment by JosebaNr** [comment]
> > > Need to add domain/subdomain name https://xyz.xyz.com
> > 
> > 
> > You mean that domain/subdomain properties are required to have a working environment? Doesn't it work by omitting that property and using local IP addresses? I want to setup a local development environment to develop html5 client changes and I don't really need nor want to register a FQDN to work locally.
> 
> You can. But i was faced the same issue in local. when i use aws ec2 machine and configure domain with ssl certificate then bbb work properly my end

Indeed, I just tried and installing it with a domain and SSL seems to work. Thanks.
